app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt

```diff
@@ -58,6 +58,12 @@ class DetailsFragment : BottomSheetDialogFragment() {
             binding.profileTxt.text = args.name
             binding.str.text = "${setTimeString(args.shr)}:${setTimeString(args.smin)}"
             binding.end.text = "${setTimeString(args.ehr)}:${setTimeString(args.emin)}"
+            binding.profileNote.text = args.notes
+            if (binding.profileNote.text != "") {
```

Comment:
Change to binding.profileNote.visibility = if (binding.profileNote.text.isBlank())...

Reply:
@rob729 Done